doc/gpumd/input_parameters/ensemble_msst.rst

```diff
@@ -0,0 +1,28 @@
+.. _kw_ensemble_msst:
+
+:attr:`ensemble` (MSST)
+=======================
+
+This keyword is used to set up a Multi-Scale Shock Technique (MSST) integrator. Please check [Reed2003]_ for more details.
+
+Syntax
+------
+
+The parameters can be specified as follows::
+
+    ensemble msst <direction> <shock_velocity> qmass <qmass_value> mu <mu_value> tscale <tscale_value>
+
+- :attr:`<direction>`: The direction of the shock wave. It can be ``x``, ``y``, or ``z``.
+- :attr:`<shock_velocity>`: The shock velocity of the shock wave in km/s.
+- :attr:`<qmass_value>`: The mass of the simulation cell. It affects the compression speed.
```

Comment:
what are the units for qmass? and also mu?

Reply:
I've added the units.